src/Commands/Handler/Internal/WorldTransferHandler.cs

```diff
@@ -30,6 +33,15 @@ protected override void Handle(WorldTransferCommand command)
 
                     SaveHelpers.LoadLevel(command.World);
 
```

Comment:
Note after our discussion: Why is this whole command even in a new Thread? Possibly risking thread safety when accessing the multiplayer manager.

Reply:
I removed the thread because it's no longer necessary